### DISCUSSION TITLE
Unable to specify start time past 24:00:00

### DISCUSSION THREAD

**Discussion by Tydoki** [MacielG1/VodRecovery#7] [General]
Specifying a start time past 24:00:00 doesn't seem to work. 
If you do this it just opens ts files to the end of the m3u8 file and says it wrote an empty file.

URL I'm using: https://streamscharts.com/channels/ariasaki/streams/43993814043

**Comment by MacielG1** [comment]
I'll try to implement a solution for this in the next update. I was taking a look at this issue, and it appears that ffmpeg doesn't handle long m3u8 videos well. By the way, this stream is still up on Twitch, so you could've used the direct URL from Twitch instead of Streamscharts. I'll let you know when I update it.

**Comment by MacielG1** [comment]
@Tydoki https://github.com/MacielG1/VodRecovery/releases/tag/1.2.7

**Reply by Tydoki** [reply]
> By the way, this stream is still up on Twitch, so you could've used the direct URL from Twitch instead of Streamscharts.

Yeah I just use Streamscharts for everything cause I know everything will be there. 

> @Tydoki https://github.com/MacielG1/VodRecovery/releases/tag/1.2.7

Thank you <3
